packages/powerapps-project-webresource/plopfile.js

```diff
@@ -12,15 +12,16 @@ module.exports = function (plop) {
 
             if (config.resourceType === 'HTML') {
                 if (file.webResources != null) {
+                    // where does filename come from?                    
                     file.webResources.push(
                         {
-                            path: `./lib/${filename}.html`,
+                            path: `./lib/${answers.entries.filename}.html`,
```

Comment:
It should be answers.filename. Also need to update line 207 to be filename instead of entity